e2e/pnpm_workspace_deps/MODULE.bazel

```diff
@@ -16,6 +16,7 @@ node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
 
 node.toolchain(
     name = "nodejs",
+    node_version = "16.9.0",
```

Comment:
I had to make the version the same as in MODULE.bazel in rules_js due to this bug which needs to be fixed upstream.

https://github.com/bazelbuild/rules_nodejs/pull/3626

Reply:
that bug fix is released, want to update this PR?

Reply:
Yup just publishing the new rnj version to bcr.